Automattic/jetpack
# Clean `_inc/` dir

This seeks to remove unused code from the `_inc/` directory.  

To test: 
- Make sure everything works in the old settings views at `wp-admin/admin.php?page=jetpack_modules`
- Make sure no console errors are shown anywhere in the app.
